programmers/완주하지-못한-선수.js

```diff
@@ -0,0 +1,16 @@
+function solution(participant, completion) {
+    const temp = {};
+    
+    for (let i of participant) {
+        if (i in temp)  temp[i] += 1; 
+        else temp[i] = 1;
+    }
+    
+    for (let j of completion) {
+         temp[j] -= 1;
+    }
+    
+    for (let i of participant) {
+        if (temp[i] === 1) return i;
+    }
+}
```

Comment:
잘했는데 for...of문은 value를 순회함. 따라서 i라는 변수 이름은 인덱스를 의미하기 때문에 옳지 못함(j, k도 마찬가지) i, j, k를 쓴다면 for...in문을 사용
```suggestion
function solution(participant, completion) {
    const temp = {};
    
    for (let i of participant) {
        if (i in temp)  temp[i] += 1; 
        else temp[i] = 1;
    }
    
    for (let j of completion) {
         temp[j] -= 1;
    }
    
    for (let i of participant) {
        if (temp[i] === 1) return i;
    }
}
```